core/src/components/radio/radio.tsx

```diff
@@ -79,6 +79,7 @@ export class Radio implements ComponentInterface {
   /** @internal */
   @Method()
   async setFocus(ev: any) {
+    // TODO(FW-2832): type (using Event triggers a build error)
```

Comment:
This is happening because we have `import { Event } from '@stencil/core'` so the compiler likely thinks you are trying to use that instead of the TypeScript `Event`. I wonder if we could alias the `Event` import to avoid this?

Reply:
Aliasing the Stencil import feels strange to me since it also changes the decorator name, but only for this file 🤔 I'll just add more details to the comment and we can revisit later.